### PR TITLE
6pm 3 office hours member route

### PR DESCRIPTION
**Pull request by m-cabello**
Implemented a office hour member route that look up the current user's email, find only the office hours that have an email for that user and return only those.
This can be used to populate the office hours in a OfficeHoursSelector component on the TutorNotesForm for entering new office hours.

### CODE REVIEW

**Review by arnav-n — APPROVED** [review]
LGTM

**Review by pconrad — DISMISSED** [review]
LGTM !

**Review by Zwang1680 — DISMISSED** [review]
Looks good from the office hours

File: src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java [hunk 1]
```diff
@@ -124,6 +124,16 @@
         return ResponseEntity.ok().body(body);
     }
 
+    @GetMapping(value = "/api/member/officeHours", produces = "application/json")
```
Comment (JacquelineMai) [inline src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java:127]: Should this endpoint also have a check that returns an unauthorized status when the user is not a member?

**Review by JacquelineMai — DISMISSED** [review]

File: src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java (hunk 1 above)
Comment (pconrad) [reply src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java:127]: Good point.... I'm going to have them do that now.

**Review by Zwang1680 — APPROVED** [review]
Looks good from office hours

File: src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java (hunk 1 above)
Comment (pconrad) [reply src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java:127]: They did it :-)

**Review by arnav-n — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**